tools/mo/openvino/tools/mo/moc_frontend/check_config.py

```diff
@@ -33,11 +51,16 @@ def legacy_extensions_used(argv: argparse.Namespace):
 
 def new_extensions_used(argv: argparse.Namespace):
     if any_extensions_used(argv):
-        extensions = argv.extensions.split(',')
+        extensions = argv.extensions
+        if not isinstance(extensions, list):
+            extensions = [extensions]
         new_ext_counter = 0
-        for extension in argv.extensions.split(','):
-            path = Path(extension)
-            if path.is_file() and (path.suffix == '.so' or path.suffix == '.dll'):
+        for extension in extensions:
+            if isinstance(extension, str):
+                path = Path(extension)
+                if path.is_file() and (path.suffix == '.so' or path.suffix == '.dll'):
+                    new_ext_counter += 1
+            else:
                 new_ext_counter += 1
```

Comment:
why do increment the counter if the new extension path is not in a string format? Please clarify

Reply:
Or should we check type of extension? I think you anticipate OpenVINO extension object here, am I correct?

Reply:
Here we consider any non-string object as extension of new type. And the exact type is checked in frontend by load_extension() method. So no need to check it here.